package.json

```diff
@@ -15,47 +15,42 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "lint": "eslint --ext .jsx,.js,.ts,.tsx .",
-    "fixcode": "eslint --ext .jsx,.js,.ts,.tsx . --fix",
+    "lint": "eslint --max-warnings 0 .",
+    "fixcode": "eslint . --fix",
     "git-hook": "yarn -s lint",
     "ncu-all": "ncu -i",
     "compile": "tsc",
     "prepare": "husky install && yarn compile",
     "sanity": "yarn lint && yarn test && tsc --noEmit && yarn outdated && echo 'success'",
     "semantic-release": "semantic-release",
-    "test": "jest"
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
   },
   "engines": {
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "eslint-config-standard-with-typescript": "^39.1.1",
-    "eslint-import-resolver-typescript": "^3.6.1"
-  },
-  "peerDependencies": {
+    "@eslint-community/eslint-plugin-eslint-comments": "^4.1.0",
     "@typescript-eslint/eslint-plugin": "^6.10.0",
     "@typescript-eslint/parser": "^6.10.0",
     "eslint": "^8.53.0",
-    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-config-standard-with-typescript": "^39.1.1",
+    "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-jest": "^27.6.0",
     "eslint-plugin-n": "^16.3.0",
     "eslint-plugin-promise": "^6.1.1",
-    "jest": "^29.7.0"
+    "globals": "^13.23.0"
+  },
+  "peerDependencies": {
+    "@txo-peer-dep/log": "^4.0.2"
```

Comment:
let's remove this completely, because upgrade will trigger fix release, and here we are not using the log at all so far.